test/integration/test-commands-test-cases.yaml

```diff
@@ -0,0 +1,38 @@
+config:
+  inherit-env: true
+
+tests:
+  001 - test login of default app (partially successful):
+    #Terminate early because command will otherwise wait for in-browser authorization stage
+    command: timeout 2s  auth0 test login $(./test/integration/scripts/get-default-app-id.sh) --no-input 2>&1
```

Comment:
Admittedly, looks odd. What's happening is running the `auth0 test login` command but enforcing a 2s timeout and piping stderr into stdout. Otherwise, it will wait indefinitely for the user to authorize in the broswer. By terminating early, this test only verifies the first stage, but its better than nothing.

Reply:
Is `timeout` a default command on MacOS? I don't appear to have it, maybe I need to install coreutils or something (I think that's fine though, maybe just needs a contributing note?)